waku/node/v2/waku_types.nim

```diff
@@ -3,14 +3,17 @@
 ## TODO Move more common data types here
 
 import
-  std/tables,
+  std/[tables, sha1],
+  times,
```

Comment:
```suggestion
  std/[tables, times],
```
Not necessary and more of a nitpick but something we've recently started doing in the Nimbus team is put `std/` in front of all stdlib imports, that way it is immediately visible which imports are from the stdlib. 
Also, I guess sha1 can be dropped here?